twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/channelIntre.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
```

Comment:
You have an issue in the file name "channelIntre".
Can you change it to "ChannelIntermediate" (and note the uppercase on first letter)

Reply:
Done